tools/reviewer/web/src/app/dashboard/file-changes/code-changes/services/ui.service.ts

```diff
@@ -1,6 +1,10 @@
 import { Injectable } from '@angular/core';
 
-import { BlockIndex, BlockLine, ChangesLine } from '../code-changes.interface';
+import {
+  BlockIndex,
+  BlockLine,
+  ChangesLine,
+} from '../code-changes.interface';
 
 @Injectable()
 export class UIService {
```

Comment:
maybe change the name for this service to something more related to text and lines. UI would mean that it does something for User Interface.

Reply:
Yeah, I was thinking about it too.

Reply:
> is `highlightjs.scss` used all across the app?
Yes! It's part of gasd. It create span tags while run-time, so the styles can be part of component.